docs/UserGuide.md

```diff
@@ -52,35 +53,57 @@ It is optimized for CLI users so that destinations and details can be added fast
     * [Sort](#sort)
         1. [Sorting by cost (L)](#1-sorting-by-cost-l)
         2. [Sorting by date (L)](#2-sorting-by-date-l)
-        3. [Sorting by importance (L)](#3-sorting-by-importance-l)
-        4. [Sorting by name (L)](#4-sorting-by-name-l)
+        3. [Sorting by datetime (L)](#3-sorting-by-datetime-l)
+        4. [Sorting by importance (L)](#4-sorting-by-importance-l)
+        5. [Sorting by name (L)](#5-sorting-by-name-l)
+        6. [Sorting by passport (L)](#6-sorting-by-passport-l)
+        7. [Sorting by mobile (L)](#7-sorting-by-mobile-l)
     * [Copy](#copy)
         1. [Copying an activity (L)](#1-copying-an-activity-l)
     * [Move](#move)
         1. [Moving an activity (L)](#1-moving-an-activity-l)
+    * [Clear](#clear)
+        1. [Clearing the data (G)](#1-clearing-the-data-g)
     * [Help](#help)
         1. [Refer to UserGuide (G)](#1-refer-to-userguide-g)
     * [Exit](#exit)
         1. [Exiting the application (G)](#1-exiting-the-application-g)
-* [FAQ \[Coming soon\]](#faq)
+* [FAQ](#faq)
 
 --------------------------------------------------------------------------------------------------------------------
 
 ## Quick Start 
 
 1. Ensure you have Java `11` or above installed in your computer.
-2. Download the latest `Wanderlust.jar` [here](https://github.com/AY2021S1-CS2103-T14-3/tp/releases/tag/v1.3)
+2. Download the latest `Wanderlust.jar` [here](https://github.com/AY2021S1-CS2103-T14-3/tp/releases)
 3. Copy the file to the folder you want to use as the _home folder_ for Wanderlust
-4. Double click to start the app.
+4. Run java -jar Wanderlust.jar in the command prompt to start the application
 5. Type the command in the command box and press Enter to execute. You can find the list of commands available in the [Command Summary](#command-summary)
 
 ![Ui](images/Ui.png)
+--------------------------------------------------------------------------------------------------------------------
+
+## Tutorial Guide
+
+Here are the steps you can follow to get familiar with _Wanderlust_
+
+Command | Description
+------------ | -------------
+`clear` | Remove all preset data from Wanderlust
+`add -travelplan n/Singapore sd/2021-05-05 ed/2021-06-06` | Adds a Singapore travelPlan to the travelPlan list 
+`goto -travelplan 1` | Go to the Singapore travelplan
+`add -friend n/Tom p/S1234567 m/12345678` | Add a friend name `Tom` with passport number `S1234567` and mobile number `12345678`
```

Comment:
The passport and mobile field needs to be changed, to reflect the new Regex. 
Passport: [STFG] + 7 numbers + [A-Z]
Mobile: [89] + 7 numbers